shared/teams/team/settings-tab/retention/index.js

```diff
@@ -166,7 +166,7 @@ class RetentionPicker extends React.Component<Props, State> {
         />
         <Box style={headingStyle}>
           <Text type="BodySmallSemibold">Message deletion</Text>
-          <Icon type="iconfont-timer" style={{marginLeft: globalMargins.xtiny}} fontSize={16} />
+          <Icon type="iconfont-timer" style={iconDeletionStyle} />
```

Comment:
@cecileboucheron we set fontsize as a prop on icons now `fontSize={isMobile ? x : y}`